### ISSUE TITLE
Budget_Summary_BP_All

### ISSUE THREAD

**Issue by AVHuntley** [afs1jes/Clarins-CA-IR-Migration#38]

**Comment by FrederickGiroux** [comment]
Could you copy 2019 to 2021? Its missing in this report

**Comment by FrederickGiroux** [comment]
Apparently the data for 2019 to 2021 is there. It doesnt load in this report tho

**Comment by A3Huntley** [comment]
Discovered issue with cascaded sheets when running for [INACTIVE BRANDS]

![Image](https://user-images.githubusercontent.com/88676502/217685300-c05ba55d-f565-40e9-a12c-5959afcf4877.png)

The member name ([47 - Annick Goutal -Soin /Skin Care]) is contains 35 characters and one illegal character (a forward slash / ).

However, the only issue seems to be that the sheet is named "Report (2)."

![Image](https://user-images.githubusercontent.com/88676502/217686127-70d7c253-5145-47d2-9cd2-fa37375a39f7.png)

One solution is to rename the dimension member.  Alternatively, ignore the issue unless it starts causing bigger problems.

**Comment by A3Huntley** [comment]
Not sure if data was missing before, or if maybe it was just hidden behind the column grouping. 2019 and 2021 seems to contain data now, unless I'm misunderstanding.


![Image](https://user-images.githubusercontent.com/88676502/217686459-40196b5e-5996-4ff8-a70a-aa6a078ea673.png)

**Comment by FrederickGiroux** [comment]
Indeed, the data is there now. The formating is off though, some totals needs to be in a darker shade of blue, some leaf level variance cell are red, and the spacing is too big on some column

**Comment by A3Huntley** [comment]
I've reset the formatting on this template by copying it over again from Prod. Some format types seem to have not made it over the last time, which was leading to differently colored rollup rows. That should be better now.

**Comment by FrederickGiroux** [comment]
Missing freezing panes

**Comment by FrederickGiroux** [comment]
Still some differences in grouping (left is dev). This example is in the Clarins summary sheet

**Comment by FrederickGiroux** [comment]
![Image](https://user-images.githubusercontent.com/124090756/234079959-d1189b70-a526-43db-9c65-203ebac43483.png)

**Comment by A3Huntley** [comment]
The change to the dim table I noted in the other "budget_summary" template should fix the issue on this template as well.

**Comment by FrederickGiroux** [comment]
Everything is ok. Thanks! :)
